libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -96,7 +96,7 @@ const static uint32_t   setcode_ram_bytes_multiplier       = 10;     ///< multip
 
 const static uint32_t   hashing_checktime_block_size       = 10*1024;  /// call checktime from hashing intrinsic once per this number of bytes
 
-const static eosio::chain::wasm_interface::vm_type default_wasm_runtime = eosio::chain::wasm_interface::vm_type::wabt;
+const static eosio::chain::wasm_interface::vm_type default_wasm_runtime = eosio::chain::wasm_interface::vm_type::eos_vm_jit;
```

Comment:
Should be based on the availability of EOS VM JIT -- should default to EOS VM when JIT isn't available (ARM builds, etc)

Reply:
:man_facepalming: sorry I remember now that neither EOS VM JIT nor EOS VM work on anything but x86_64, so academic for now